pkg/api/graph/graph.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type Node struct {
-	concrete.Node
+	graph.Node
```

Comment:
I thought it was needed for something I was working on a while ago, but it's not - I can undo it.